.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+- env:
+  - CGO_ENABLED=0
+  goos:
+    - freebsd
+    - openbsd
+    - solaris
+    - windows
+    - linux
+    - darwin
+  goarch:
+    - amd64
+    - '386'
+    - arm
+    - arm64
+  ignore:
+    - goos: darwin
+      goarch: '386'
+    - goos: darwin
+      goarch: arm
+    - goos: darwin
+      goarch: arm64
+    - goos: openbsd
+      goarch: arm
+    - goos: openbsd
+      goarch: arm64
+    - goos: solaris
+      goarch: arm
+    - goos: solaris
+      goarch: arm64
+    - goos: solaris
+      goarch: '386'
+    - goos: windows
+      goarch: arm
+    - goos: windows
+      goarch: arm64
+  binary: '{{ .ProjectName }}_{{ .Tag }}'
+archives:
+- format: zip
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+checksum:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_SHA256SUMS'
+  algorithm: sha256
+signs:
+  - artifacts: checksum
+    args:
+      - "--local-user"
+      - "51852D87348FFC4C" # Replace this with your GPG signing key ID
```

Comment:
Should we just include a bad string here to force the issue?

Reply:
It would result in an error if users don't have the private key part of this. I'd like to leave it here to eventually publish this provider so that people can see the example docs correlating to the published HTML on the Registry, including a lot of formatting examples.